src/main/java/woowacourse/auth/ui/AuthenticationInterceptor.java

```diff
@@ -16,6 +17,10 @@ public AuthenticationInterceptor(JwtTokenProvider jwtTokenProvider) {
 
     @Override
     public boolean preHandle(HttpServletRequest request, HttpServletResponse response, Object handler) {
+        if (HttpMethod.OPTIONS.matches(request.getMethod())) {
```

Comment:
```suggestion
        if (CorsUtils.isPreFlightRequest(request)) {
```
Options 의 모든 메서드에 대해서 로직을 수행하기보다는 Preflight로 특정하면 좀 더 CORS 관련 처리라는 걸 보여줄 수 있을 것 같네요. 혹시 모를 Cors 요청이 아닌 Options 메서드에 대해서도 인증을 수행할 수 있구요.